web/src/lib/components/faces-page/merge-face-selector.svelte

```diff
@@ -69,8 +69,9 @@
         message: `Merged ${count} ${count === 1 ? 'person' : 'people'}`,
         type: NotificationType.Info,
       });
+      people = people.filter((person) => !results.some((result) => result.id === person.id && result.success === true));
       await invalidateAll();
-      onClose();
+      onMerge();
```

Comment:
If we are not using `onMerge` anywhere else, maybe we can just use `dispatch()` here?